api-tests/features/p2-strategic-app.feature

```diff
@@ -0,0 +1,68 @@
+@core-back-api-test
```

Comment:
This tag is different to other tests, which have `@Build`. What was the reason for this?

Reply:
@MikeCollingwood  I believe these were changed to so we can run the tag  @core-back-api-test

Reply:
I misunderstood another PR I saw - reverted